README.md

```diff
@@ -87,13 +85,15 @@ flowchart LR;
 In order to use this workflow, you must already have completed the following steps:
 
 1. Created a Synapse project shared with challenge participants.
+1. Grant the DPE team `Administrator` privileges to said project.
 1. Created an evaluation queue within the Synapse project.
```

Comment:
May want to add that DPE team needs to be granted to the evaluation queue as well

Reply:
+1